jobs/rocoto/ocnanalletkf.sh

```diff
@@ -18,6 +18,6 @@ export PYTHONPATH
 
 ###############################################################
 # Execute the JJOB
-"${HOMEgfs}/jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_LETKF"
+"${HOMEgfs}/jobs/JGDAS_GLOBAL_MARINE_ANALYSIS_LETKF"
```

Comment:
Since this job will be run in both early and late cycle, I suggest:
```suggestion
"${HOMEgfs}/jobs/JGLOBAL_MARINE_ANALYSIS_LETKF"
```